Classes/ApplicationInsights.h

```diff
@@ -14,7 +14,7 @@
 // Notification message which MSAIApplicationInsights is listening to, to retry requesting updated from the server
 #define MSAINetworkDidBecomeReachableNotification @"MSAINetworkDidBecomeReachable"
 
-#define MSAI_SERVER_URL   @"https://dc.services.visualstudio.com/v2/track"
+#define MSAI_SERVER_URL   @"https://eus-breeziest-in.cloudapp.net/v2/track"
```

Comment:
Why this change? The URLs should effectively point to the same server but I don't know if the latter one is guaranteed to be stable and stay the same.